### ISSUE TITLE
Confusing error if old dmp files are around

### ISSUE THREAD

**Issue by dschwoerer** [boutproject/xBOUT#235]
Running with a lower number of processes results in this error:
```
ValueError: Each run directory does not contain an equal number of output files. If the parallelization scheme of your simulation changed partway-through, then please load each directory separately and concatenate them along the time dimension with xarray.concat().
```

I am only loading one simulation, so this error does not seem to be correct.

Preferably I would have old data just ignored, as it is with `collect`:
https://github.com/boutproject/boutdata/blob/master/boutdata/collect.py#L279
https://github.com/boutproject/boutdata/blob/master/boutdata/collect.py#L310

Would you be happy for a PR?

**Comment by johnomotani** [comment]
Yes, PR would be welcome. This case was not considered when the existing error message was written!

**Comment by dschwoerer** [comment]
Hmm, changing the error is fairly easy, but ignoring files without breaking the current behaviour is I guess essentially impossible.

If we could assume just a single simulation per folder, that would make things easier.
But right now we support `BOUT.dmp.{0,1}.nc` to be two different simulations. Would it be ok to break this behaviour?
